src/main/java/frc/robot/commands/elevator/MoveToHeight.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.commands.elevator;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Elevator;
+
+public class MoveToHeight extends CommandBase {
+  Elevator m_elevator; 
+  double m_targetHeight; 
+  /**
+   * This command allows the elvator to move to the heights 
+   * of the bottom, middle, and top nodes, or any height that we choose. 
```

Comment:
you could robobly just say that this command moves the elevator to the given height ranther than stating that it can go to certain nodes. This could be ok if you plann to emplment  a node paramater in wich case add a TODO